packages/app-admin/src/components/SplitView/SplitView.tsx

```diff
@@ -26,7 +26,7 @@ const grid = css({
 const RightPanelWrapper = styled("div")({
     backgroundColor: "var(--mdc-theme-background)",
     overflow: "auto",
-    height: "calc(100vh - 64px)"
+    height: "calc(100vh - 45px)"
```

Comment:
Shall we create a variable to hold this value, as shown below, and use it instead of the arbitrary value?

```
module.exports = {
  theme: {
    extend: {
      spacing: {
        'header-bar': '45px',
      }
    }
  }
}
```

This is merely a suggestion; let me know what you think and if this makes sense.

Reply:
Your proposal makes sense. But, what it does not make so much sense is this whole approach I'd say.

I think having this kind of calculation should be avoided, and we should find another way to ensure the content below header fills the whole remaining vert space of the screen. 

All in all, I believe there must be a better way to accomplish this. I'll check it out in my upcoming PR.